app/models/dialog_field.rb

```diff
@@ -13,6 +13,7 @@ class DialogField < ApplicationRecord
                                   :message => "Field Name %{value} is reserved."}
 
   default_value_for :required, false
+  default_value_for :visible, true
```

Comment:
I think there's also a Base#reset_column_information that would accomplish the same thing.